tests/tests.py

```diff
@@ -13,14 +17,24 @@
                      cached, cached_view, cached_as, cached_view_as
 from cacheops import invalidate_fragment
 from cacheops.templatetags.cacheops import register
+from cacheops.transaction import _function_queue, in_transaction
+
 decorator_tag = register.decorator_tag
 from .models import *
 
 
-class BaseTestCase(TestCase):
+# django.test.TestCase wraps individual tests in atomic blocks. Old tests were not written to check
+#  the change in functionality cacheop's transaction support implements when in atomic blocks.
+#  Changing to django.test.TransactionTestCase fixes this issue. Tests are not in atomic blocks.
+#  This does make tests run slower then before, since tables are truncated and fixtures reapplied
+#  between tests.
+class BaseTestCase(TransactionTestCase):
     def setUp(self):
         super(BaseTestCase, self).setUp()
         invalidate_all()
+        # for mysql, django will add the query 'SET SQL_AUTO_IS_NULL = 0' before the first query
+        #  run a query before our tests in order to trigger this before we start counting queries.
+        Category.objects.nocache().exists()
```

Comment:
This is related to switching to `TransactionTestCase`. With `TestCase` every test wrapped in `atomic()`, the first line goes to SQL always as `BEGIN TRANSACTION` which django prepends with `SET SQL_AUTO_IS_NULL = 0'`.